framework/src/main/java/org/apache/drill/test/framework/TestCaseModeler.java

```diff
@@ -86,8 +86,8 @@ public static TestCaseModeler createTestCase(String testId, String type,
     public String outputFormat;
     @JsonProperty("expected-file")
     public String expectedFile;
-    public String username = TestDriver.drillProperties.get("USERNAME");
-    public String password = TestDriver.drillProperties.get("PASSWORD");
+    public String username = Utils.drillProperties.get("USERNAME");
+    public String password = Utils.drillProperties.get("PASSWORD");
     @JsonProperty("verification-type")
```

Comment:
TestDriver used to hold a copy of drillProperties which was obtained from Utils but not anymore. We should go to source directly anyways.